Add two notes about the job (#94)
2i2c-org/2i2c-brochure: content/jobs/2022/product-community-lead.md

* Experience **using an open source language** in a research/education setting. 
* An interest in learning about **workflows with the open source language Python**, and open source workflows around it for research and education.
* Experience or strong interest in **cloud infrastructure and data workflows** for research or education, and how this can lead to more collaboration, openness, and productivity (for example, via managed services and [Analysis Ready, Cloud Optimized datasets](https://eartharxiv.org/repository/view/2726/)).
* Experience with **distributed and complex open source communities** and ability to communicate and collaborate effectively with these communities.

## A note on scope and expectations

We know that this role is very broad in scope, and that there is a lot of room for interpretation of the specific responsibilities and duties. This is somewhat intentional, and somewhat unavoidable, given that this is the first role of its kind within 2i2c.

> We recognize that one risk in these situations is to have unrealistic expectations, or an unsustainable amount of work for the person involved. We'll do everything we can to set these expectations intentionally, and to refine them as we learn more, to ensure that this role is sustainable.

Author: ```suggestion
We recognize that one risk in these situations is to have unrealistic expectations, or an unsustainable amount of work for the person involved. We'll do everything we can to set these expectations intentionally, and to refine them as we learn more, to ensure that this role is sustainable. The person in this role will need to be comfortable with a larger-than-normal amount of ambiguity and flexibility, though part of their job will be to _reduce_ this ambiguity for themselves and for others in the future.
```

> We recognize that one risk in these situations is to have unrealistic expectations, or an unsustainable amount of work for the person involved. We'll do everything we can to set these expectations intentionally, and to refine them as we learn more, to ensure that this role is sustainable. The person in this role will need to be comfortable with a larger-than-normal amount of ambiguity and flexibility, though part of their job will be to _reduce_ this ambiguity for themselves and for others in the future.

Reviewer: I recognize the intention to be explicit about the risk involved but IMHO being that explicitness goes against the job posting attractiveness. 
I would try to be more straightforward, instead. I mean... the first sentence looks great (line 90) and then I will just follow up just with "We'll do everything we can to set these expectations intentionally, and to refine them as we learn more, to _reduce_ this ambiguity, to ensure that this role is sustainable."
Just my totally subjective point of view 😉.

> We know that this role is very broad in scope, and that there is a lot of room for interpretation of the specific responsibilities and duties. This is somewhat intentional, and somewhat unavoidable, given that this is the first role of its kind within 2i2c.
> 
> We recognize that one risk in these situations is to have unrealistic expectations, or an unsustainable amount of work for the person involved. We'll do everything we can to set these expectations intentionally, and to refine them as we learn more, to ensure that this role is sustainable. The person in this role will need to be comfortable with a larger-than-normal amount of ambiguity and flexibility, though part of their job will be to _reduce_ this ambiguity for themselves and for others in the future.

Author: ```suggestion
We know that this role is very broad in scope, and that there is a lot of room for interpretation of the specific responsibilities and duties. This is somewhat intentional, and somewhat unavoidable, given that this is the first role of its kind within 2i2c. We will do everything we can to set these expectations intentionally, and to refine them as we learn more, in order to reduce this ambiguity and ensure that this role is sustainable.
```

* Experience **using an open source language** in a research/education setting. 
* An interest in learning about **workflows with the open source language Python**, and open source workflows around it for research and education.
* Experience or strong interest in **cloud infrastructure and data workflows** for research or education, and how this can lead to more collaboration, openness, and productivity (for example, via managed services and [Analysis Ready, Cloud Optimized datasets](https://eartharxiv.org/repository/view/2726/)).
* Experience with **distributed and complex open source communities** and ability to communicate and collaborate effectively with these communities.

## A note on scope and expectations

> We know that this role is very broad in scope, and that there is a lot of room for interpretation of the specific responsibilities and duties. This is somewhat intentional, and somewhat unavoidable, given that this is the first role of its kind within 2i2c. We will do everything we can to set these expectations intentionally, and to refine them as we learn more, in order to reduce this ambiguity and ensure that this role is sustainable.

Author: What do you think @damianavila - good point about not under-selling ourselves :-)

Reviewer: Super! Thanks!
